### PR TITLE
SW-5122 Implement ToDo list with mock data

### PR DESCRIPTION
**Pull request by nickgraz**
There are several things that might change once the back end is done, but the UI is done. Those things will be addressed later.

Will need a new component in `web-components` for fluid truncated text.

### CODE REVIEW

**Comment by nickgraz** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/2519?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#2519** <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/2519?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#2518** <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/2518?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#2516** <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/2516?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#2515** <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/2515?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#2514** <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/2514?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#2513** <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/2513?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @nickgraz and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: src/scenes/Home/ParticipantHomeView/ToDoSection.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  return (
+    <Box marginTop={theme.spacing(2)}>
+      <Typography fontSize={'20px'} fontWeight={600} lineHeight={'28px'}>
+        {section}
+      </Typography>
+      <Box
+        sx={{
+          borderBottom: `1px solid ${theme.palette.TwClrBaseGray100}`,
+          height: '1px',
+          width: '100%',
```
Comment (karthikbtf) [inline src/scenes/Home/ParticipantHomeView/ToDoSection.tsx:27]: try flex props?

File: src/scenes/Home/ParticipantHomeView/index.tsx [hunk 2]
```diff
@@ -6,24 +6,30 @@ import DismissibleWrapper from 'src/components/common/DismissibleWrapper';
...
+import ToDo from './ToDo';
 import WelcomeBanner from './WelcomeBanner';
 
 const ParticipantHomeView = () => {
   const theme = useTheme();
 
   return (
     <PageWithModuleTimeline>
-      <Grid display={'flex'} flexDirection={'column'}>
+      <Grid display={'flex'} flexDirection={'column'} width={'100%'}>
```
Comment (karthikbtf) [inline src/scenes/Home/ParticipantHomeView/index.tsx:17]: maybe flexGrow or xs={12} ?

**Review by karthikbtf — APPROVED** [review]
Nicely done, like the library of little components. Some comments about width styling.

File: src/scenes/Home/ParticipantHomeView/index.tsx [hunk 3]
```diff
@@ -6,24 +6,30 @@ import DismissibleWrapper from 'src/components/common/DismissibleWrapper';
...
+import ToDo from './ToDo';
 import WelcomeBanner from './WelcomeBanner';
 
 const ParticipantHomeView = () => {
   const theme = useTheme();
 
   return (
     <PageWithModuleTimeline>
-      <Grid display={'flex'} flexDirection={'column'}>
+      <Grid display={'flex'} flexDirection={'column'} flexGrow={1}>
```
Comment (karthikbtf) [inline src/scenes/Home/ParticipantHomeView/index.tsx:17]: 💯 🙇

**Review by karthikbtf — APPROVED** [review]
Thank you!!

File: src/scenes/Home/ParticipantHomeView/index.tsx (hunk 3 above)
Comment (nickgraz) [reply src/scenes/Home/ParticipantHomeView/index.tsx:17]: It was your idea! 🙇 🙇 🙇

**Comment by nickgraz** [comment]
### Merge activity

* **Apr 16, 7:05 PM EDT**: @nickgraz started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/terraware/terraware-web/2519).

**Pull request merged**